examples/gallery/mvdr/mvdr_tutorial.py

```diff
@@ -0,0 +1,291 @@
+"""
+MVDR with torchaudio
+====================
+
+**Author** `Zhaoheng Ni <zni@fb.com>`__
+
+"""
+
+######################################################################
+# Overview
+# ========
+# 
+# This is a tutorial on how to apply MVDR beamforming by using `torchaudio <https://github.com/pytorch/audio>`__.
+# 
+# Steps
+# 
+# - Ideal Ratio Mask (IRM) is generated by dividing the clean/noise
+#   magnitude by the mixture magnitude.
+# - We test all three solutions (``ref_channel``, ``stv_evd``, ``stv_power``)
+#   of torchaudio's MVDR module.
+# - We test the single-channel and multi-channel masks for MVDR beamforming.
+#   The multi-channel mask is averaged along channel dimension when computing
+#   the covariance matrices of speech and noise, respectively.
+
+
+######################################################################
+# Preparation
+# ===========
+# 
+# First, we import the necessary packages and retrieve the data.
+# 
+# The multi-channel audio example is selected from
+# `ConferencingSpeech <https://github.com/ConferencingSpeech/ConferencingSpeech2021>`__
+# dataset.
+# 
+# The original filename is
+# 
+#    ``SSB07200001\#noise-sound-bible-0038\#7.86_6.16_3.00_3.14_4.84_134.5285_191.7899_0.4735\#15217\#25.16333303751458\#0.2101221178590021.wav``
```

Comment:
@nateanl I want to make sure that we list all the usages of licensed materials.
Can you check that I listed all the original materials bellow?

I did not get what the remaining numbers mean, and I am guessing that they are some augmentation parameters and not another data. 

`7.86_6.16_3.00_3.14_4.84_134.5285_191.7899_0.4735`
`15217`
`25.16333303751458`
`0.2101221178590021`

Reply:
According to the [mixture method](https://github.com/ConferencingSpeech/ConferencingSpeech2021/blob/master/simulation/mix_wav.py#L330),
```python
utt_id = clean + seg + noise + seg + rir + seg + start_time + seg + snr + seg + scale + '.wav'
```
``seg`` is  "\#"
``7.86_6.16_3.00_3.14_4.84_134.5285_191.7899_0.4735`` is the rir file name, the rir is simulated by the image method.
``15217`` is the start_time
``25.16333303751458`` is the snr
``0.2101221178590021`` is the scale